ITE/5/README.adoc

```diff
@@ -0,0 +1,153 @@
+= ITE-5: Replace signature envelope with SSL signing spec
+:source-highlighter: pygments
+:toc: preamble
+:toclevels: 2
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="2"]
+|===
+| ITE
+| 5
+
+| Title
+| Replace signature envelope with SSL signing spec
+
+| Sponsor
+| link:https://github.com/santiagotorres[Santiago Torres]
+
+| Status
+| Draft 💬
+
+
+| Type
+| Standards
+
+| Created
+| 2020-09-28
+
+|===
+
+[[abstract]]
+== Abstract
+
+This link:https://github.com/in-toto/ITE[in-toto enhancement (ITE)] proposes
+removing the in-toto signature scheme requirements, and allow users to adopt the
+signature envelope that better fits their ecosystem. This has the following
```

Comment:
This ITE doesn't seem to actually talk about how that will work - that is, how it will support arbitrary envelope types. Should we instead focus this ITE purely on supporting SSLSS (we need a better name!) and talk about how that will be implemented in in-toto?

Reply:
(This is a dupe of the other thread. Let's continue discussion there.)